### ISSUE TITLE
soft drop factor

### ISSUE THREAD

**Issue by weareveryawesom** [tetrio/issues#825]
### Checklist

- [X] I've **checked both [open and recently closed issues](https://github.com/tetrio/issues/issues?q=is%3Aissue)** to ensure this feature is not already requested.
- [X] I am using the **newest version** of TETR.IO.
- [X] I am certain this is a **feature request**, not a bug.
- [X] I've **thought this feature through** properly.

### Is your feature request related to a problem? Please describe.

This request is not related to a problem. I have been playing jstris for quite a while and I normally use softdrop not as instant but very fast because that's how I have always played. I felt like the gap between X40 and infinite was a little too big.

### Describe the solution you'd like

I was wondering if you guys can add more numbers to the soft drop factor such as x50 or even x60. i just feel like x40 is too slow yet instant softdrop is too fast for me.

### Describe alternatives you've considered

just don't set the limit to the number that players can plug in?

**Comment by ZaptorZap** [comment]
As an ∞ SDF user, I personally can't really see the appeal in Jstris's "ultra" soft drop factor... I have to practically prepare to press the soft drop button and release it within the six frame window it gives me. At that point, you're basically using ∞ SDF with extra delays for no advantage, since stopping midway through is very inconsistent.

https://user-images.githubusercontent.com/64891705/142789960-1507bbc6-0d50-4184-a3d4-ab30eb7a24ff.mp4

Mind you this IS a 144fps recording, without any speed effects applied to it. That window, 6 frames in a 144FPS scenario, is unimaginably small... the only benefit to it is a visual appeal for your mental image of a piece soft dropping instead of instantly dropping, perhaps? TETR.IO's equivalent 40X SDF takes about 23 frames instead of 6, for comparison. 

Furthermore, regarding that alternative, that of unlimited SDF values; I find it rather unlikely for a couple reasons. First off, there's still gotta be a slider with a defined maximum ∞ value -- this "slider design scheme" just doesn't work if there's no maximum. You may have noticed it's possible to select the  SDF number itself already, and type in your own value! To that I bring up another issue: the possibility of users defining absolutely ridiculous values and forcing the server to validate, say, TETRA LEAGUE's initial natural gravity value of `0.02` times a user's SDF value of `875953532.3789575632` *every 1/60th of a second* while `soft drop` is held. I might be overstating the amount of problems a simple multiplication problem can cause for any computer, but, still, internet users will be internet users and they would abuse that lack of a limit no matter how obscure "higher values" are. 
![image](https://user-images.githubusercontent.com/64891705/142791090-223a174d-7bdf-4ba1-88e7-664467d966b8.png)

**Comment by mat1jaczyyy** [comment]
i mean, the slider can become more of an exp/log curve which would make more sense and allow accessing the 40-70 range but with much less precision.

jstris applies softdrop per frame, and if you are running on 144fps vsync your softdrops will be 2.4x faster than a "regular" user running 60fps vsync even when using the same settings.

also, while it's extremely hard to release the SD key in 6/144 frames (41ms), the issue is more about being unable to do a rotation input quickly to keep it fluid, but you rather have to introduce a delay if you want to keep the visual animation. this is especially more noticeable in tetrio's 23/144 (160ms) window. 

also, mind you that zeph is known for playing consistently at DAS values as low as 2.5F (42ms), so it's definitely possible for some people to consistently press keys in shorter intervals than that.

**Comment by weareveryawesom** [comment]
yes, I use 60fps but I think fps doesn't really matter in this case because
I have tried on both. I just feel like you are making excuses. I do
understand why tetr can't put in the system where people can put in values,
but I do feel like you can increase the factor up to some point such as x50
or even x60. I don't see a reason why not to. Also just because you use
infinite sdf doesn't mean that you can just assume that people who doesn't
just infinite sdf handling the fast soft drop
